src/Utils/getDropDownItems.js

```diff
@@ -0,0 +1,17 @@
+import { get } from 'lodash';
```

Comment:
imo we should have next structure for this `src/common/utils/dropdown.js`? mb dropdownUtils

@aliaksei-chumakou @AndreiShumski2 what do you think?

Reply:
moved to suggested place